_test/test_data_loaders/test_rundata.m

```diff
@@ -364,12 +364,12 @@ function test_set_field_have_preference(obj)
             run=set_lattice_field(run,'psi',20,'-ifempty');
             assertEqual(10,get_rundata(run,'psi'));
 
-            lat=oriented_lattice();
+            lat=oriented_lattice('psi',10);
             run.lattice = lat;
-            assertEqual(0,get_rundata(run,'psi'));
+            assertEqual(10,get_rundata(run,'psi'));
 
             run=set_lattice_field(run,'psi',20,'-ifempty');
-            assertEqual(20,get_rundata(run,'psi'));
+            assertEqual(10,get_rundata(run,'psi'));
```

Comment:
because it sets it only if it has not been already set. Dodgy concept but was here before. Come from the fact that sometimes psi is not written into nxspe file, but if it was written, its probably more accurate then what you want to set up.  

May be should be removed all together.  Too complicated to be useful -- old code